components/form/InputFormControl.js

```diff
@@ -1,25 +1,43 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-function InputFormControl({ id, label, placeholder, type, value, onChange }) {
+function InputFormControl({
+  error,
+  id,
+  label,
+  placeholder,
+  required = false,
+  type,
+  value,
+  onChange,
+}) {
   const handleChange = (event) => onChange(event.target.value);
 
   return (
     <div className="form-control">
       {label && (
         <label className="label" htmlFor={id}>
-          <span className="label-text">{label}</span>
+          <span className="label-text">{`${label}${
+            required ? ' *' : ''
+          }`}</span>
```

Comment:
There's a way to do this in CSS: https://stackoverflow.com/questions/11197671/use-css-to-automatically-add-required-field-asterisk-to-form-inputs

Earlier I was thinking that a CSS `:after` solution might be the way to go, but now I'm sort of on the fence. @marekrozmus which way do you think is cleaner code, and why? @theflucs I'd be curious to hear your thoughts too.

Reply:
done - please check if you like the solution :)